test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java

```diff
@@ -1361,6 +1361,8 @@ public void indexRandom(boolean forceRefresh, boolean dummyDocuments, boolean ma
         final List<Exception> actualErrors = new ArrayList<>();
         for (Tuple<IndexRequestBuilder, Exception> tuple : errors) {
             if (ExceptionsHelper.unwrapCause(tuple.v2()) instanceof EsRejectedExecutionException) {
+                // temporarily logging errors to aid debugging https://github.com/elastic/elasticsearch/issues/43144
+                logger.info("Error indexing, reindexing doc");
```

Comment:
Maybe it would it make sense to add more details (like the message) of the exception we unwrapped here to get even more info about whats happening.
I think it makes sense to have this logging in general (not only to investigate 43144) but I'm wondering if it would make sense to set this to a lower level (e.g. Debug) and enable it on the test we are investigating in 43144 using the new `@TestIssueLogging` annotation that is just introduced in https://github.com/elastic/elasticsearch/pull/44477). Might be worth waiting for that to be merged, otherwise you could also use the existing `@TestLogging` annotation to set the level to DEBUG for the test logging in `testQuotedQueryStringWithBoost()`.